src/test/kotlin/blackJack/domain/PlayersTest.kt

```diff
@@ -0,0 +1,23 @@
+package blackJack.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class PlayersTest {
+    @Test
+    fun makePlayers() {
+        val players = Players(listOf(Player("name"), Player("second")))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        val players = Players("name", "second")
```